src/components/Projects/HoldingsProjects.tsx

```diff
@@ -17,70 +17,67 @@ export default function HoldingsProjects() {
     theme: { colors },
   } = useContext(ThemeContext)
 
+  if (!userAddress) {
+    return (
+      <div
+        style={{
+          textAlign: 'center',
+          padding: 20,
+        }}
+        hidden={isLoading}
+      >
+        <Trans>Connect your wallet to see your holdings.</Trans>
+      </div>
+    )
+  }
+
   return (
-    <React.Fragment>
-      {!userAddress ? (
-        <div
-          style={{
-            textAlign: 'center',
-            padding: 20,
-          }}
-          hidden={isLoading}
-        >
-          <Trans>Connect your wallet to see your holdings.</Trans>
-        </div>
-      ) : (
-        <React.Fragment>
-          {projects && projects.length > 0 && (
-            <React.Fragment>
-              <Grid>
-                {projects.map(p => (
-                  <ProjectCard project={p} />
-                ))}
-              </Grid>
-            </React.Fragment>
-          )}
+    <>
+      {projects && projects.length > 0 && (
```

Comment:
I'm a big fan of `array.length` sans >0, how do you guys usually operate? The performance diff are severely marginal, but I know when I read `array.length` it means array has something. When parsing `projects.length > 0` I have to do one extra cognitive step, if that makes sense. I don't really mind either, but it's good for consistency (my vote is on sans > 0 though ;) )

Reply:
I don't feel strongly about this, although there is a slight distinction between them in React-land: https://reactjs.org/docs/jsx-in-depth.html?#booleans-null-and-undefined-are-ignored. 

A quick grep of the codebase reveals we use both approaches fairly equally. I'd generally agree with you but in JSX, I think it's safer to opt for the strict boolean eval offered by `> 0`.

In any case, I'll leave this as-is for now.